xpriv/xpriv.go

```diff
@@ -1,14 +1,19 @@
 // Package xpriv manges keys
 package xpriv
 
+// "github.com/libsv/go-bk/bip39" - no replacements
+
 import (
 	"fmt"
 
-	"github.com/libsv/go-bk/bip32"
+	bip32 "github.com/bitcoin-sv/go-sdk/compat/bip32"
+	chaincfg "github.com/bitcoin-sv/go-sdk/transaction/chaincfg"
+
 	"github.com/libsv/go-bk/bip39"
-	"github.com/libsv/go-bk/chaincfg"
 )
 
+// TODO: "github.com/libsv/go-bk/bip39" - no replacement (GenerateEntropy, Mnemonic, MnemonicToSeed)
```

Comment:
I think you are right, I will pass it to Deggen. Those seems missing

Reply:
this was added to go-sdk and the added functions were used